Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -440,6 +440,27 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
             }
         }
 
+        if let expandVariableFromTarget = scheme.runAction?.expandVariableFromTarget {
+            guard let graphTarget = graphTraverser.target(
+                path: expandVariableFromTarget.projectPath,
+                name: expandVariableFromTarget.name
+            ) else { return nil }
+
+            guard let buildableReference = try createBuildableReference(
+                graphTarget: graphTarget,
+                graphTraverser: graphTraverser,
+                rootPath: rootPath,
+                generatedProjects: generatedProjects
+            ) else { return nil }
```

Comment:
Are we sure we want to return nil and skip the rest of the method?
Can't we just "merge" these check in the `if let expandVariableFromTarget …`?